pkg/operation/botanist/resource_manager.go

```diff
@@ -62,7 +62,9 @@ func (b *Botanist) DeleteManagedResources(ctx context.Context) error {
 func (b *Botanist) WaitUntilManagedResourcesDeleted(ctx context.Context) error {
 	return retry.Until(ctx, 5*time.Second, func(ctx context.Context) (done bool, err error) {
 		managedResources := &resourcesv1alpha1.ManagedResourceList{}
-		if err := b.K8sSeedClient.Client().List(ctx, managedResources, client.InNamespace(b.Shoot.SeedNamespace), client.MatchingLabels(map[string]string{ManagedResourceLabelKeyOrigin: ManagedResourceLabelValueGardener})); err != nil {
+		if err := b.K8sSeedClient.Client().List(ctx, managedResources,
```

Comment:
```suggestion
		if err := b.K8sSeedClient.Client().List(ctx,
		managedResources,
```

If you want to have a newline for the arguments then do it for every argument :)